nx-X11/programs/Xserver/hw/nxagent/NXcomposite.c

```diff
@@ -1,359 +0,0 @@
-#ifndef NXAGENT_UPGRADE
```

Comment:
The code in NXcomposite.c is also embraced by this:

```
#ifndef NXAGENT_UPGRADE
[...]
#endif /* #ifndef NXAGENT_UPGRADE */
```

So, removing it is sane here, IMHO.